src/DataValues/DecimalValue.php

```diff
@@ -58,7 +58,9 @@ public function __construct( $value ) {
 		}
 
 		$value = trim( $value );
-
+		if ( $value !== '' && $value[0] !== '-' && $value[0] !== '+' ) {
```

Comment:
I'm not sure I understand. The clause has specific check for $value[0] != '-' - could you explain how would it add + to -1?